lib/quby/questionnaires/entities/question.rb

```diff
@@ -166,9 +166,6 @@ def initialize(key, options = {})
             @validations << {type: :valid_float, explanation: options[:error_explanation]}
           elsif @type == :integer
             @validations << {type: :valid_integer, explanation: options[:error_explanation]}
-          elsif @type == :date
```

Comment:
De validatie is afhankelijk van de components-optie.

In principe zou je de componenten ook kunnen opvragen in `Question#initialize` met options[:components], maar ik vind het netter om zoveel mogelijk de componenten als attribuut (`question.components`) op te vragen.

Vandaar dat ik het zo heb gebouwd.